docs/using/index.rst

```diff
@@ -15,4 +15,18 @@ the results. Finally, we will show how to use the ``queue`` command to interact
 
    codeless
    running
-   queueing
\ No newline at end of file
+   queueing
+
+
+.. _using_components:
```

Comment:
I think this should either have its own index or be a sub-section as it comes up as a section now. Here: https://juaml.github.io/junifer/pr-preview/pr-174/using/index.html, you have the section entry and also have a separate outermost entry in ToC.

Reply:
should be a subheading